File: src/com/krishna/miniproject/Book.java
package com.krishna.miniproject;

public class Book {
    private String author;
    private String title;
    private String book_id;
    private boolean isAvailable=true;

    public Book(String author, String title, String book_id) {
        this.author = author;
        this.title = title;
        this.isAvailable = true;
        this.book_id = book_id;
    }

//    Methods:
//getTitle(), getAuthor(), getBookId()
//isAvailable()
//issueBook() → sets isAvailable = false
//returnBook() → sets isAvailable = true
    public String getAuthor() {
        return author;
    }

    public String getTitle(){
        return title;
    }

    public String getBookId(){
        return book_id;
    }
    public boolean isAvailable(){
        return isAvailable;
    }
    public void issueBook(){
        isAvailable=false;
    }
    public void returnBook(){
        isAvailable=true;
    }

}

File: Dart/src/krishna/miniproject/Book.dart
class Book{

  String _title;
  String _author;
  String _bookId;
  bool _isAvailable=true;

  Book(this._title, this._author, this._bookId);

  String get title => _title;
  String get author => _author;
  String get bookId => _bookId;
  bool get isAvailable => _isAvailable;

  void issueBook(){
   _isAvailable=false;
  }
  void returnBook(){
    _isAvailable=true;
  }



}
